### ISSUE TITLE
Deepfry built in

### ISSUE THREAD

**Issue by mistyhands** [adekmaulana/caligo#181]
Hi, I noticed this user bot uses my deep fryer ;) I don't really mind, but it seems really popular so my server gets a lot of load from all these clients. I can help you build in the deep frying if you want so it doesn't have to call my bot. Thanks

**Comment by tomyprs** [comment]
Btw, caligo doesn't have a deep fryer, but have a glitch, or maybe this is same? , Ah also, feel free to contribute here, This will really help the lazy dev @adekmaulana .

**Comment by mistyhands** [comment]
Sorry, you're right. There was a past bot of his with a deep fryer that used mine but I got confused. It spams my VPS with load

**Comment by tomyprs** [comment]
> Sorry, you're right. There was a past bot of his with a deep fryer that used mine but I got confused. It spams my VPS with load

You're Right. and the plugin isn't working as it should now, if you can, make a PR, and add credits before the license header, or maybe it will be added to the reademe later.
THANKS YOU. :-)

Warm Regards,

TOMI PRASETIO.
Developer Experience Specialist
USERBOTINDO COMMUNITY Support

**Comment by adekmaulana** [comment]
Hi! sorry for late reply, my past bot is already dead long ago. Probably some user forked the repo and update it.
